public/jsx/views/card-listing.jsx

```diff
@@ -23,25 +23,16 @@ module.exports = React.createClass({
   },
 
   componentDidMount: function() {
-    $.ajax({
-      method: 'get',
-      url: '/api/braintree/mozilla/paymethod/',
-      context: this,
-    }).then(function(data) {
-      if (this.isMounted()) {
-        // Ignore react/no-did-mount-set-state eslint warning
-        if (data.length) {
-          console.log('Card data found, showing card form');
-          this.setState({cards: data}); // eslint-disable-line
-        } else {
-          console.log('No card data found, showing card form');
-          this.transitionTo('card-form');
-        }
-      }
-    }).fail(function() {
-      // TODO: some error state.
-      console.log('Card list retrieval failed');
-    });
+    var user = UserStore.getLoggedInUser();
```

Comment:
I'm not sure yet. Right now it will blow up on the console which is ok. Once we wire up full screen errors (https://github.com/mozilla/payments/issues/70) we'll need to think about some kind of global exception handler. It would be annoying to have to catch every error but maybe we'll have to do that because javascript.